pkg/client/doc.go

```diff
@@ -0,0 +1,62 @@
+/*
+Package client provides Kubernetes client implementation.
+
+Client implements methods to perform CRUD operations on Kubernetes objects in a cluster.
+
+Create, Get, Update, Delete and List Kubernetes Objects:
+
+	// use helper methods to build config for client.
```

Comment:
Isn't this 100% redundant with your example?

Reply:
Yes. I wanted to see in godoc if the `Example` shows up in overview or not. Will take it out if it does.